src/pages/docs/step-ca/provisioners.mdx

```diff
@@ -150,43 +150,55 @@ These commands require you to sign in as an Admin user.
 
 ### Enable Remote Provisioner Management
 
-To enable remote provisioner management, update your `"authority"` block in `ca.json` to include the following:
+#### On a New CA
 
-```json
-"authority": {
-		"enableAdmin": true,
-		"provisioners": []
-},
-```
+To enable remote provisioner management on a new CA, pass `--remote-management` to `step ca init`. This will:
 
-The `"provisioners"` list must be empty, and `"enableAdmin"` must be set to `true`.
+* Add an initial [JWK](#jwk) provisioner called Admin JWK to the database.
+* Create an initial Super Admin user, with default username `step`, and link it to the Admin JWK provisioner.
 
-### Bootstrap Remote Provisioner Management
+#### On an Existing CA
 
-During this step:
-* An initial [JWK](#jwk) provisioner called Admin JWK will be added to the database.
-* An initial Super Admin user, with username `step`, will be created and linked to the Admin JWK provisioner.
-
-"Admins" can remotely manage provisioners.
-"Super Admins" are Admins that can also manage the list of Admins for the CA.
+To migrate your `ca.json` provisioners to the database, and to enable remote provisioner management on an existing CA:
 
 1. Stop `step-ca` if it is running.
-2. Start `step-ca` from the command line.
+2. Update your `"authority"` block in `ca.json` to include the following:
+
+  ```json
+  "authority": {
+  		"enableAdmin": true,
+          "provisioners": [
+            ...
+          ]
```

Comment:
The indent is off. Is that fixed when the site is generated? If not, it needs a fix here